Add max chip amount

Fueredoriku opened the pull request:
- Add visual feedback for the max cap

- Add chip sprite to bidsigns for consistency

toberge reviewed (changes requested):
Seems to work as intended, though the color choice isn't ideal. It shouldn't be pure #ff0000 >:(, try using a color that is in use elsewhere in the game. And perhaps show 20 (MAX) instead of just MAX on the chip popup.

![image](https://github.com/hackerspace-ntnu/Red-Planet-Rampage/assets/44986729/a9390f0b-1da1-4414-bc0b-fcff29dae5d0)

toberge commented on Assets/Scripts/UI/PlayerHUDController.cs:
```diff
@@ -207,8 +207,11 @@ public void OnChipChange(int amount)
-        chipAmount.text = amount.ToString();
+        bool isMax = amount == PlayerIdentity.MaxChips;
+        Debug.Log(PlayerIdentity.MaxChips);
+        Debug.Log(amount);
```
Please remove these log statements :)))))

Fueredoriku replied:
Fixed 👍

Fueredoriku committed:
Add max chip amount #723

- Add visual feedback for the max cap

- Add chip sprite to bidsigns for consistency

toberge reviewed (approved):
Looks like a better choice of red 👍
